components/infobox/wikis/valorant/infobox_person_player_custom.lua

```diff
@@ -164,12 +172,12 @@ function CustomPlayer:getPersonType(args)
 	local roleData = _ROLES[(args.role or ''):lower()]
 	if roleData then
 		if roleData.staff then
-			return {store = 'Staff', category = 'Staff'}
+			return {store = 'staff', category = 'Staff'}
 		elseif roleData.talent then
-			return {store = 'Talent', category = 'Talent'}
+			return {store = 'talent', category = 'Talent'}
 		end
 	end
-	return {store = 'Player', category = 'Player'}
+	return {store = 'player', category = 'Player'}
```

Comment:
sure about this?
and why?

Reply:
According to @Rathoz they needed to be stored without uppercasing to make sure the PlayerIntroduction module worked

Reply:
okay

Reply:
> sure about this? and why?

PlayerIntro expects lowercase
Types are generally stored with lowercase, see eg. https://github.com/Liquipedia/Lua-Modules/blob/main/components/squad/commons/squad_row.lua#L23-L24